aws_auth.tf

```diff
@@ -8,7 +8,7 @@ resource "null_resource" "update_config_map_aws_auth" {
   depends_on = ["aws_eks_cluster.this"]
 
   provisioner "local-exec" {
-    command     = "for i in {1..5}; do kubectl apply -f ${var.config_output_path}config-map-aws-auth_${var.cluster_name}.yaml --kubeconfig ${var.config_output_path}kubeconfig_${var.cluster_name} && break || sleep 10; done"
+    command     = "for i in `seq 1 10`; do kubectl apply -f ${var.config_output_path}config-map-aws-auth_${var.cluster_name}.yaml --kubeconfig ${var.config_output_path}kubeconfig_${var.cluster_name} && exit 0 || sleep 10; done; exit 1"
```

Comment:
Does this work on Windows @RothAndrew ?

Reply:
It's still a shell specific command, so it won't work with `cmd`. Windows users have to use something like Git Bash through the use of the `local_exec_interpreter` variable. This does not break that functionality.

Reply:
Ahh, I see